### ISSUE TITLE
FIRApp.configure() in init function stuck the app on logo page

### ISSUE THREAD

**Issue by shabib3** [EddyVerbruggen/nativescript-plugin-firebase#607]
I am trying to run my app on IOS but it stucks on the logo page.
after I debugged it I found out the is fails on the `firebase.init()` function, the specific problematic line in the `init` function is `FIRApp.configure();`.

I tried to re-add the `GoogleService-Info.plist` file into `app/App_Resources/iOS/`, but it did nothing.

I am running the `firebase.init()` function in `main.ts`.
I tried also to require `firebase` in `main.ts` and run the `init` function in my first page.

I am using NS Angular, and firebase plugin version 5.1.2.

Anyone have any idea what can it be?

**Comment by sivsushruth** [comment]
Event I am facing the same issue. How was this fixed @shabib3 ?

**Comment by shabib3** [comment]
I think the demo folder show how it should be.

this is what I am doing:

main.ts:
```
import { platformNativeScriptDynamic } from "nativescript-angular/platform";
import { AppModule } from "./app.module";

import * as firebase from 'nativescript-plugin-firebase';

platformNativeScriptDynamic().bootstrapModule(AppModule);
```

app.component.ts:
```
export class AppComponent {
  constructor() {
    firebase.init({
      ...
      ...
    })
      .then(() => console.log(">>>>> Firebase initialized"))
      .catch(err => console.log(">>>>> Firebase init error: " + err));
  }
}
```

**Comment by sivsushruth** [comment]
Got it, trying it now. I was always putting it inside ngOnInit as specified in docs

**Comment by sivsushruth** [comment]
Even this is crashing. FIRApp.configure() just crashes the iOS app without error @shabib3

**Comment by sivsushruth** [comment]
Okay, have figured out the fix

Maybe it can help the plugin developers

So, build with xcode 10 fails
I reverted xcode to 9.4 and things are working ok now.

**Comment by craguila** [comment]
I ran into the same issue but using nativescript-vue.
Building with xcode 10.2.1 the app crashes on splash screen.
Debugging I found the error was also in FIRApp.configure().
I have found in [https://firebase.google.com/docs/reference/swift/naming-migration-guide](https://firebase.google.com/docs/reference/swift/naming-migration-guide)
that somethings have been renamed.
After building the proyect went to vendor.js and replaced all occurrences of FIRApp with FirebaseApp. Then the vendor.js file was transfered to device and worked fine. 
I think some things should be updated on the plugin.
